## Test Coverage Report
The overall test coverage is **$83.3%**.

### Coverage Details
```
GoExperiments/channels_experiments/channels.go:8: DoChannelTests 0.0% GoExperiments/channels_experiments/channels.go:12: SendMessage 0.0% GoExperiments/channels_experiments/channels.go:17: ReceiveMessage 0.0% GoExperiments/channels_experiments/channels.go:23: ForwardMessage 0.0% GoExperiments/channels_experiments/channels.go:29: TestChannelDirections 0.0% GoExperiments/channels_experiments/channels.go:40: SendMessageToChannelWithNoReceiver 90.0% GoExperiments/errors_experiments/ErrorHandlingAs.go:8: RunErrorsAsTests 0.0% GoExperiments/errors_experiments/ErrorHandlingAs.go:15: runErrorsAs 0.0% GoExperiments/errors_experiments/ErrorHandlingAs.go:39: errorAsMyWonderfulValueErrorByValueType 100.0% GoExperiments/errors_experiments/ErrorHandlingAs.go:51: errorAsMyWonderfulValueErrorByPointerType 100.0% GoExperiments/errors_experiments/ErrorHandlingAs.go:66: errorAsMyWonderfulPointerErrorByPointerType 75.0% GoExperiments/errors_experiments/ErrorHandlingAs.go:90: errorAsMyWonderfulPointerErrorByPointerToPointerType 100.0% GoExperiments/errors_experiments/error_wrapping.go:11: Error 100.0% GoExperiments/errors_experiments/error_wrapping.go:15: Unwrap 100.0% GoExperiments/errors_experiments/error_wrapping.go:19: MakeCupOfCoffee 100.0% GoExperiments/errors_experiments/error_wrapping.go:44: Error 100.0% GoExperiments/errors_experiments/error_wrapping.go:54: Error 100.0% GoExperiments/errors_experiments/error_wrapping.go:61: Is 75.0% GoExperiments/errors_experiments/error_wrapping.go:69: BoilWater 100.0% GoExperiments/errors_experiments/error_wrapping.go:89: Error 100.0% GoExperiments/errors_experiments/error_wrapping.go:94: AddCoffeeToWater 100.0% GoExperiments/errors_experiments/errortypes.go:12: Error 100.0% GoExperiments/errors_experiments/errortypes.go:21: Error 100.0% GoExperiments/errors_experiments/errortypes.go:26: giveMeMyError 0.0% GoExperiments/iterations_range_fun/range_stuf.go:8: SumOverSliceInt 100.0% GoExperiments/iterations_range_fun/range_stuf.go:19: AddNumbersFromStartToEnd 100.0% GoExperiments/iterations_range_fun/range_stuf.go:27: GetMapDictionaryContainKeyValueIteration 100.0% GoExperiments/iterations_range_fun/range_stuf.go:36: GetMapDictionaryContainKeyValueLookup 100.0% GoExperiments/main.go:3: main 0.0% GoExperiments/slices_fun/slices.go:7: GetLastElementInSlice 100.0% GoExperiments/slices_fun/slices.go:15: GetFirstElementInSlice 100.0% GoExperiments/slices_fun/slices.go:23: GetSubsetOfSlice 100.0% GoExperiments/slices_fun/slices.go:39: GetSliceSplitInHalf 100.0% GoExperiments/slices_fun/slices.go:54: ChuckSlice 100.0% GoExperiments/slices_fun/slices.go:75: AppendSliceToSlice 100.0% GoExperiments/slices_fun/slices.go:82: AppendVariadicValueToSlice 100.0% GoExperiments/slices_fun/slices.go:87: AppendValueToSlice 100.0% GoExperiments/structs_services/InMemoryJokes.go:12: NewInMemoryJokeService 100.0% GoExperiments/structs_services/InMemoryJokes.go:25: GiveMeAJoke 100.0% GoExperiments/structs_services/InMemoryJokes.go:31: GiveMeJokeAtIndex 100.0% GoExperiments/structs_services/InMemoryJokes.go:44: RemoveJokeAtIndex 71.4% GoExperiments/structs_services/InMemoryJokes.go:58: JokeCount 100.0% GoExperiments/structs_services/InMemoryJokes.go:63: AddJoke 100.0% GoExperiments/structs_services/InMemoryJokes.go:68: removeIndexFromSlice 100.0% GoExperiments/structs_services/JokeService.go:15: NewJoker 0.0% GoExperiments/structs_services/JokeService.go:21: AddJoke 100.0% GoExperiments/structs_services/JokeService.go:25: JokeCount 0.0% GoExperiments/structs_services/StringJokeService.go:17: NewStringJokesService 100.0% GoExperiments/structs_services/StringJokeService.go:36: GiveMeAJoke 83.3% GoExperiments/structs_services/StringJokeService.go:47: GiveMeJokeAtIndex 88.9% GoExperiments/structs_services/StringJokeService.go:65: RemoveJokeAtIndex 69.2% GoExperiments/structs_services/StringJokeService.go:89: JokeCount 75.0% GoExperiments/structs_services/StringJokeService.go:100: AddJoke 75.0% GoExperiments/structs_services/StringJokeService.go:116: GetJokeStorage 80.0% GoExperiments/structs_services/StringJokeService.go:127: SaveJokeStorage 80.0% GoExperiments/types_experiments/types_alias_fun.go:14: ReturnTheValueRepeatedValueForStringAlias 100.0% GoExperiments/types_experiments/types_alias_fun.go:18: ReturnCoordinatesUsingAlias 100.0% GoExperiments/types_experiments/types_assertion_fun.go:7: CalculateWingLoadSafeTypeAssertion 100.0% GoExperiments/types_experiments/types_assertion_fun.go:32: CalculateWingLoadSwitchTypeAssertion 100.0% GoExperiments/types_experiments/types_assertion_fun.go:54: CalculateWingLoadNotSafeTypeAssertion 100.0% GoExperiments/types_experiments/types_definition_fun.go:14: ReturnCoordinatesUsingDefinition 100.0% GoExperiments/types_experiments/types_definition_fun.go:18: ReturnTheValueRepeatedValueForStringDefinition 100.0% GoExperiments/types_experiments/types_embedding_fun.go:27: CalculateWingload 100.0% GoExperiments/types_experiments/types_embedding_fun.go:32: CalculateWingload 100.0% GoExperiments/types_experiments/types_enum_fun.go:22: GetManufacturerAbbreviation 100.0% GoExperiments/types_experiments/types_enum_fun.go:30: GetManufacturerPriority 100.0% GoExperiments/types_experiments/types_func_fun.go:5: VerifyJumperCanJump 100.0% GoExperiments/types_experiments/types_func_fun.go:10: NotAllowedToJump 100.0% GoExperiments/types_experiments/types_func_fun.go:13: AllowedToJump 100.0% GoExperiments/types_experiments/types_func_fun.go:17: DFURuleset 93.3% GoExperiments/types_experiments/types_func_handler_experiment.go:20: Write 0.0% GoExperiments/types_experiments/types_func_handler_experiment.go:30: GoOn 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:35: AddTest 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:46: AddABC 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:55: AddXXX 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:64: HandlePayload 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:72: CreateInvocation 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:76: AddMiddleware 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:81: AddInitialHandler 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:90: EndHandler 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:95: Handle 100.0% GoExperiments/types_experiments/types_func_handler_experiment.go:113: Invoke 100.0% total: (statements) 83.3%
```

--- a/.github/workflows/go.yml
+++ b/.github/workflows/go.yml
@@ -57,7 +57,7 @@ jobs:
       uses: peter-evans/create-or-update-comment@v4
       with:
         token: ${{ secrets.PAT_TOKEN }}
-        issue-number: ${{ github.event.pull_request.number }}'
+        issue-number: ${{ github.event.pull_request.number }}
         edit-mode: append
         body: |
           ## Test Coverage Report

--- a/.github/workflows/go.yml
+++ b/.github/workflows/go.yml
@@ -58,7 +58,7 @@ jobs:
       with:
         token: ${{ secrets.PAT_TOKEN }}
         issue-number: ${{ github.event.pull_request.number }}'
-        edit-mode: replace
+        edit-mode: append
         body: |
           ## Test Coverage Report
           The overall test coverage is **$${{ env.coverage }}**.

--- a/.github/workflows/go.yml
+++ b/.github/workflows/go.yml
@@ -33,17 +33,16 @@ jobs:
         go tool cover -func=coverage.out > coverage.txt
         
         coverage=$(tail -n 1 coverage.txt  | awk '{print $3}')
-        coverage_details=$(cat coverage.txt)
+        
         echo "Overall test coverage: $coverage"
-        echo "Detailed test coverage: coverage_details"
         
         echo "setting coverage brief"
         echo "coverage=$coverage" >> $GITHUB_ENV
+
         echo "Setting details"
-        
         {
           echo 'coverage_details<<EOF'
-          echo $coverage_details
+          cat coverage.txt
           echo EOF
         } >> "$GITHUB_ENV"
         
@@ -58,7 +57,8 @@ jobs:
       uses: peter-evans/create-or-update-comment@v4
       with:
         token: ${{ secrets.PAT_TOKEN }}
-        issue-number: ${{ github.event.pull_request.number }}
+        issue-number: ${{ github.event.pull_request.number }}'
+        edit-mode: replace
         body: |
           ## Test Coverage Report
           The overall test coverage is **$${{ env.coverage }}**.

